build.gradle

```diff
@@ -59,6 +59,7 @@ dependencies {
     testImplementation "org.testcontainers:testcontainers:1.14.2"
     testImplementation group: 'org.awaitility', name: 'awaitility', version: '4.0.3'
     testCompile("org.assertj:assertj-core:3.16.1")
+    testImplementation 'org.awaitility:awaitility:4.0.3'
```

Comment:
it's already there in line 60:
testImplementation group: 'org.awaitility', name: 'awaitility', version: '4.0.3'